cvat-ui/src/actions/requests-actions.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2024 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { ActionUnion, createAction } from 'utils/redux';
+import { RequestsQuery } from 'reducers';
+import { Request, InstanceType, getCore } from 'cvat-core-wrapper';
+
+const core = getCore();
+
+export enum RequestsActionsTypes {
+    GET_REQUESTS = 'GET_REQUESTS',
+    GET_REQUESTS_SUCCESS = 'GET_REQUESTS_SUCCESS',
+    GET_REQUESTS_FAILED = 'GET_REQUESTS_FAILED',
+    GET_REQUEST_STATUS_SUCCESS = 'GET_REQUEST_STATUS_SUCCESS',
+    GET_REQUEST_STATUS_FAILED = 'GET_REQUEST_STATUS_FAILED',
+    REQUEST_FINISHED = 'REQUEST_FINISHED',
+    REQUEST_FAILED = 'REQUEST_FAILED',
+    CANCEL_REQUEST = 'CANCEL_REQUEST',
+    CANCEL_REQUEST_FAILED = 'CANCEL_REQUEST_FAILED',
+    DELETE_REQUEST = 'DELETE_REQUEST',
+    DELETE_REQUEST_FAILED = 'DELETE_REQUEST_FAILED',
+}
+
+export const requestsActions = {
+    getRequests: (query?: RequestsQuery) => createAction(RequestsActionsTypes.GET_REQUESTS, { query }),
+    requestFinished: (request: Request) => createAction(RequestsActionsTypes.REQUEST_FINISHED, { request }),
+    requestFailed: (request: Request) => createAction(RequestsActionsTypes.REQUEST_FAILED, { request }),
+    getRequestsSuccess: (requests: Request[]) => createAction(
```

Comment:
Request[] is a wrong type, as you read property `count`, not existing on type Array

Correct one is `Awaited<ReturnType<typeof core['requests']['list']>>`